conf/machine/imx8mmevk.conf

```diff
@@ -39,7 +39,7 @@ SPL_BINARY = "spl/u-boot-spl.bin"
 DDR_FIRMWARE_NAME = "lpddr4_pmu_train_1d_imem.bin lpddr4_pmu_train_1d_dmem.bin lpddr4_pmu_train_2d_imem.bin lpddr4_pmu_train_2d_dmem.bin"
 
 # Set u-boot DTB
-UBOOT_DTB_NAME = "fsl-imx8mm-evk.dtb"
+UBOOT_DTB_NAME = "imx8mm-evk.dtb"
```

Comment:
Please squash this on the U-Boot upgrade, as it is required since its update.

Reply:
Done